samples/bluetooth/peripheral_hr/boards/siwx917_rb4338a.overlay

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2021 Cypress Semiconductor Corporation.
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+&egpio0 {
+zephyr,pm-device-runtime-auto;
+};
+ 
+&nwp {
+   power-profile = "deep-sleep-with-ram-retention";
+   status = "okay";
```

Comment:
AFAIK, `status = "okay"` is useless for now since `"silabs,siwx91x-nwp"` is not yet a true driver.